### ISSUE TITLE
XRAY - CVSS range doesn't support decimals

### ISSUE THREAD

**Issue by sottlmarek** [jfrog/terraform-provider-xray#2]
**Description**
Hello Artifactory devs, I am trying to create new policy set for Xray. I found small problem in policy definition. I am using CVSS3 scoring system and it works with decimals. 

Example: 
```
resource "artifactory_xray_policy" "my_severity_policy" {
  name        = "my-severity"
  description = "Managed by Terraform"
  type        = "security"
  rules {
    name     = "high-severity-rule"
    priority = 1
    criteria {
      cvss_range {
        from = 7
        to   = 10
      }
    }
    actions {
      fail_build = false
      mails      = []
      webhooks   = []
      block_download {
        active    = false
        unscanned = false
      }
    }
  }
}
```

**Requirements for and issue**
- [ ] I need to be able to work with decimals in code block like this: 
```
     cvss_range {
        from = 7.1
        to   = 9.4
      }
    }
```
- [ ] My version of Artifactory is  7.15.4 rev
- [ ] My version of Terraform (Terraform v1.0.4) 

**Expected behavior**
I want to be able to work with more granular ranges then whole numbers. Its possible via console but not via Terraform. 

**Additional context**
This issue affects all the companies working with proper CVSS scoring and especially with https://www.first.org/cvss/calculator/3.0 . Working with granular policies allows to setup proper blocking, notification and watcher setup for X-Ray.

**Comment by chb0github** [comment]
@danielmkn - is this resolved with the latest code?

**Comment by danielmkn** [comment]
@chb0github , yes

**Comment by danielmkn** [comment]
Released in v0.0.1
